src/Templating/src/Microsoft.DotNet.Web.Spa.ProjectTemplates/content/Angular-CSharp/ClientApp/src/app/app.component.css

```diff
@@ -1,6 +1 @@
-@media (max-width: 767px) {
-  /* On small screens, the nav menu spans the full width of the screen. Leave a space for it. */
-  .body-content {
-    padding-top: 50px;
-  }
-}
+
```

Comment:
I don't know how Angular works, but can we delete the empty file altogether?

Reply:
You need to remove it from the ts file then:

https://github.com/aspnet/AspNetCore/blob/master/src/Templating/src/Microsoft.DotNet.Web.Spa.ProjectTemplates/content/Angular-CSharp/ClientApp/src/app/app.component.ts#L6

Reply:
I can take care of this. Thanks!